src/components/tooltip/Tooltip.tsx

```diff
@@ -100,6 +100,8 @@ export type TooltipProps = {
     stopBubble?: boolean;
     /** Text to show in the tooltip */
     text?: React.ReactNode;
+    /** True when should hide tooltip (usually when another tooltip is showing) */
+    shouldHideTooltip?: boolean;
```

Comment:
do you need to add this prop? can you use an existing one instead e.g. isShown?

Reply:
Thank you for catching this, I can use isShown instead

Reply:
@DanDeMicco - ready for review again